tractor/_portal.py

```diff
@@ -102,14 +105,25 @@ async def aclose(self):
         """Cancel associated remote actor task and local memory channel
         on close.
         """
-        if self._rx_chan._closed:
+        rx_chan = self._rx_chan
+        stats = rx_chan.statistics()
+
+        if rx_chan._closed:
             log.warning(f"{self} is already closed")
             return
 
+        if stats.open_receive_channels > 1:
+            # if we've been cloned don't kill the stream
+            log.debug("there are still consumers running keeping stream alive")
+            return
+
         if self._shielded:
             log.warning(f"{self} is shielded, portal channel being kept alive")
             return
 
+        # close the local mem chan
+        rx_chan.close()
```

Comment:
This likely needs to be put above.

Thanks to @richardsheridan for catching 🏄🏼